docker/docker-compose.dev.yml

```diff
@@ -16,15 +16,14 @@ services:
       - PGUSER=postgres
       - PGPASSWORD=password
       - PGDATABASE=postgres
-      - RUDDERSTACK_WRITE_KEY="1vikeGadtB0Y0oRDFNL2Prdhkbp"
-      - RUDDERSTACK_DATA_PLANE_URL="https://analytics.lightdash.com"
+      - RUDDERSTACK_WRITE_KEY=1vikeGadtB0Y0oRDFNL2Prdhkbp
+      - RUDDERSTACK_DATA_PLANE_URL=https://analytics.lightdash.com
       - SECURE_COOKIES=false
       - LIGHTDASH_SECRET="not very secret"
       - LIGHTDASH_LOG_LEVEL=debug
-      - SENTRY_DSN=""
-      - NODE_ENV="development"
-      - LIGHTDASH_CONFIG_FILE="/usr/app/lightdash.yml"
-      - DBT_DEMO_DIR="/usr/app/examples/full-jaffle-shop-demo"
+      - NODE_ENV=development
```

Comment:
I've updated these env variables because strings go wtihout quotes. Otherwise it causes different unexpected errors like:

![Screenshot from 2023-06-16 13-47-30](https://github.com/lightdash/lightdash/assets/1983672/0365511a-2d37-4185-a528-fd6f5b29fbd2)

![Screenshot from 2023-06-16 13-47-18](https://github.com/lightdash/lightdash/assets/1983672/e5f659db-801f-4aa7-8d47-d6dc7f2ea098)